src/main/java/pl/cleankod/exchange/provider/nbp/ExchangeRatesNbpClientService.java

```diff
@@ -0,0 +1,8 @@
+package pl.cleankod.exchange.provider.nbp;
+
+import pl.cleankod.exchange.provider.nbp.model.RateWrapper;
+
+public interface ExchangeRatesNbpClientService {
```

Comment:
Same here - only one implementation. Why do we need that interface?

Reply:
see above